src/ACEtk/interpretation/DEDX1/CRTP.hpp

```diff
@@ -10,8 +10,8 @@ class CRTP{
   using TempT = decltype( 1.0 * mev );
 
   #include "ACEtk/interpretation/DEDX1/CRTP/src/outOfBounds.hpp"
-  #include "ACEtk/interpretation/DEDX1/CRTP/S1.hpp"
-  #include "ACEtk/interpretation/DEDX1/CRTP/S0.hpp"
+  //#include "ACEtk/interpretation/DEDX1/CRTP/S1.hpp"
```

Comment:
If there is a preference to remove the CPT code that doesn't work we could create a branch to explicitly capture it and then clear out whatever we aren't using. That is actually my preference.